lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7478,6 +7486,18 @@ bool importer::requiresCPlusPlus(const clang::Module *module) {
   });
 }
 
+bool importer::isCxxStdModule(const clang::Module *module) {
+  if (module->getTopLevelModuleName() == "std")
+    return true;
+  // In recent libc++ versions the module is split into multiple top-level
+  // modules (std_vector, std_utility, etc).
+  if (module->getTopLevelModule()->IsSystem &&
```

Comment:
Is the check for `IsSystem` required? The code doesn't do the same for just the `std` module.

Reply:
This check is meant to handle potential cases where a client has a non-stdlib module called, for instance, `std_extras`. It is not strictly required, but I think it reduces the possibility of treating a custom module as a stdlib module.

Reply:
Let me know if you think this is harmful, I can put up a separate patch to change this logic.